func.cc

```diff
@@ -583,7 +583,7 @@ void CallFunc(const vector<Value*>& args, Evaluator* ev, string* s) {
     Intern("5"), Intern("6"),  Intern("7"), Intern("8"), Intern("9")
   };
 
-  const string&& func_name = args[0]->Eval(ev);
+  const string&& func_name = TrimSpace(args[0]->Eval(ev)).as_string();
```

Comment:
For ckati, I'm pathologically trying to avoid string allocations. Let me remove as_string() call.